library/ssl_misc.h

```diff
@@ -1708,7 +1725,12 @@ int mbedtls_ssl_tls13_write_sig_alg_ext( mbedtls_ssl_context *ssl,
                                          unsigned char *buf,
                                          unsigned char *end,
                                          size_t *out_len);
-
+/*
+ * Parse TLS 1.3 Signature Algorithm extension
+ */
+int mbedtls_ssl_tls13_parse_sig_alg_ext( mbedtls_ssl_context *ssl,
+                                         const unsigned char *buf,
+                                         const unsigned char *end );
```

Comment:
The function should be merged into `mbedtls_ssl_parse_sig_algs` . If this PR has been rebased on top of #5355. please double check `mbedtl_ssl_parse_sig_algs`

Reply:
I would suggest to not do this in this PR. I am going to add a ticket to do that as part of the work on the server side. But in this PR it is ok to keep the TLS 1.2 / TLS 1.3 code duplication.

Reply:
I've created the issue: https://github.com/ARMmbed/mbedtls/issues/5460.